src/utils/sort.ts

```diff
@@ -83,8 +83,8 @@ export function sortRows(rows: any[], columns: any[], dirs: SortPropDir[]): any[
       const propB = valueGetter(b, prop);
 
       const comparison = cachedDir.dir !== SortDirection.desc ?
-        cachedDir.compareFn(propA, propB) :
-        -cachedDir.compareFn(propA, propB);
+        cachedDir.compareFn(propA, propB, a, b, true) :
```

Comment:
Can you add a comment here whats going on. Also, better name for a/b would be good too.

Reply:
I added some comments to the code and renamed a/b to rowA and rowB which should be less cryptic. Also I added few lines to documentation for consideration.